libs/common/src/main/kotlin/sak/Sak.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.etterlatte.libs.common.sak
+
+data class Sak(val ident: String, val sakType: String, val id: Long)
```

Comment:
hva er forskjellen på ident og id? er id:n en mer human-readable id på saken?

Reply:
Det er personen det gjelder for den gitte saken. fødselsnummer ifølge databasen. Jeg trodde det var saksbehandlerident men det gir jo ingen mening.